src/main/java/org/apache/commons/lang3/text/StrBuilder.java

```diff
@@ -95,7 +104,7 @@
     /** Internal data storage. */
     protected char[] buffer; // TODO make private?
```

Comment:
To avoid warnings later in the class, this should probably be `@SameLen("this")`. See https://checkerframework.org/manual/#index-annotating-fixed-size